build/x86_64_debian/Dockerfile

```diff
@@ -41,7 +18,7 @@ RUN apt-get -y update \
         make \
```

Comment:
FWIW, I've found that some Ruby gems might also need `cmake` and `libicu-dev` (see [`igorpeshansky-build-dockerfiles-stretch`](https://github.com/GoogleCloudPlatform/google-fluentd/tree/igorpeshansky-build-dockerfiles-stretch), which I've just rebased off `master`)... Was that a fluke? Does the build work without them?

Reply:
See the internal ticket for the related testing.